### PR TITLE
Add single post template and associated parts

### PR DESCRIPTION
**Pull request by saralohr**
Got a good start on the single post pattern + parts. Still a few TODOs in there, but seemed PR-able.

screen shot 2016-06-16 at 5 18 47 pm

cc @tylersticka @mrgerardorodriguez @erikjung

### CODE REVIEW

File: wp-content/themes/cloudfour/templates/single.twig [hunk 1]
```diff
@@ -1,30 +1,58 @@
...
-						{% include "comment.twig" with {comment:cmt} %}
-					{% endfor %}
-				</div>
-			</section>
-		{% endif %}
-		</article>
-	</div><!-- /content-wrapper -->
+{% block body %}
+  <div class="u-containProse">
+    <article class="u-spaceEnds1 u-spaceItems1 post-type-{{post.post_type}}" id="post-{{post.ID}}">
```
Comment (tylersticka) [inline wp-content/themes/cloudfour/templates/single.twig]: Is the `post-type-` class necessary?

File: wp-content/themes/cloudfour/templates/single.twig [hunk 2]
```diff
@@ -1,30 +1,58 @@
...
+
+      <header class="u-spaceItems01">
+        <h1>{{post.title}}</h1>
+        <p>
+          Written by
+          <a href="{{post.author.path}}">{{post.author.name}}</a>
+            on
+            <time datetime="{{post.post_date|date("y-m-d")}}">
+            {{post.post_date|date}}
+          </time>
```
Comment (tylersticka) [inline wp-content/themes/cloudfour/templates/single.twig]: Nit: Weird indentation in this block

**Comment by tylersticka** [comment]
Couple of comments inline. Otherwise LGTM 👍

File: wp-content/themes/cloudfour/templates/single.twig (hunk 1 above)
Comment (saralohr) [inline wp-content/themes/cloudfour/templates/single.twig]: Oh good catch, yeah I think we can drop that.

**Pull request merged**